protobufs/livekit_agent.proto

```diff
@@ -22,45 +22,46 @@ option ruby_package = "LiveKit::Proto";
 import "livekit_models.proto";
 
 service AgentService {
-  rpc CreateAgentJobDefinition(CreateAgentJobDefinitionRequest) returns (JobDefinition);
-  rpc DeleteAgentJobDefinition(DeleteAgentJobDefinitionRequest) returns (JobDefinition);
-  rpc ListAgentJobDefinitions(ListAgentJobDefinitionsRequesst) returns (ListAgentJobDefinitionsResponse);
+  rpc CreateAgentDispatch(CreateAgentDispatchRequest) returns (AgentDispatch);
```

Comment:
I'm liking Dispatch! i think it's more straight forward to understand.

nit: should we drop the `Agent` from the method name? i.e. 

```suggestion
  rpc CreateDispatch(AgentCreateDispatchRequest) returns (AgentDispatch);
```